### PR TITLE
Portuguese translation added

### PR DESCRIPTION
**Pull request by acacio90**
Add language Portuguese/BR #63

### CODE REVIEW

File: src/legacy/dev-application/languages/pt.php
Comment (nebojsatomic) [inline src/legacy/dev-application/languages/pt.php:1]: @acacio90 you used the wrong file to copy contents it seems, it should be contents of **_src/legacy/dev-application/languages/sr.php_** and then the strings should be translated.
The file that you copied here doesn't have all the strings, it is a bit confusing to have to copy sr.php, I know, but for now this is the way it is
as stated in #51

File: src/legacy/dev-application/languages/creator/pt.php
Comment (nebojsatomic) [inline src/legacy/dev-application/languages/creator/pt.php:1]: @acacio90 do not remove the copyright part at the beginning of the files, please.
the same goes for the other two files

Comment (acacio90) [reply src/legacy/dev-application/languages/creator/pt.php:1]: Ok, the header has been corrected and updated to match the same structure as sr.php.

**Comment by acacio90** [comment]
Ok, the header has been corrected and updated to match the same structure as **sr.php**.

**Comment by nebojsatomic** [comment]
@acacio90 thanks!

File: src/public/js/language/creator/pt.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-lang.Loading = "";
-lang.Working = "Trabalhando...";
-lang.AYS = "Você tem certeza?";
-lang.Done = "Concluído!";
-lang.TOpen = "O template está abrindo...";
-lang.POpen = "A página está abrindo...";
-lang.TUpdate = "Salvando template...";
-lang.PUpdate = "Salvando página...";
-lang.PageOpened = "Página aberta!";
+lang.Loading = '';
```
Comment (nebojsatomic) [inline src/public/js/language/creator/pt.js:3]: @acacio90  Could you use double quotes here please? and for every string where you used single quotes in this file, also change to double quotes.

**Comment by acacio90** [comment]
Done, @nebojsatomic thanks for the instructions.

**Review by nebojsatomic — APPROVED** [review]

**Pull request merged**